src/components/select/select.component.ts

```diff
@@ -233,7 +233,6 @@ export class SelectComponent<T> implements OnInit, OnChanges, OnDestroy, Control
             take(1),
             takeUntil(this._onDestroy))
             .subscribe(() => {
-                this.dropdownOpen = true;

```

Comment:
Hmm, I think there is another case to consider here. If I tab to the select input and type 'U' for example, the list will appear.

If I reload the page, and I tab past the select, then using tab return to the select and type 'U' the list does not appear.